src/main/java/ToDo.java

```diff
@@ -0,0 +1,10 @@
+public class ToDo extends Task {
```

Comment:
Perhaps you may want to implement a final variable `type` here as in like the other classes `Deadline` and `Event`?